index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <section class="Mobile_background">
+
+    
+        <header>
+        <img src="images/My_Logo.png" class="logo" alt="Logo">
+        <img src="images/menu.png" class="menu" alt="Menu">
+        </header>
+       <!-- Headline section-->
+        <div class="headline_section">
```

Comment:
- [ ] Please change this `div` to a more semantic tag, in this case, you should use `section` tag instead of a `div` this will give your project a better structure